swarm/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "libp2p-swarm"
 edition = "2018"
 description = "The libp2p swarm"
-version = "0.22.0"
+version = "0.22.1"
```

Comment:
I am declaring this as a non-breaking change, thus bumping the patch version only. Let me know in case you prefer a minor version bump.